test/state/roles/actions.test.js

```diff
@@ -13,15 +13,27 @@ const mockStore = configureMockStore(middlewares);
 
 jest.mock('api/roles', () => ({
   getRoles: jest.fn(),
+  getRole: jest.fn(),
   postRoles: jest.fn(),
+  putRole: jest.fn(),
 }));
 
 const GET_ROLES_PROMISE = {
   ok: true,
   json: () => new Promise(res => res({ payload: LIST_ROLES_OK })),
 };
 
+const GET_ROLE_PROMISE = {
```

Comment:
Here too, using mockApi from testUtils would allow you to do stuff like this:

`getRole.mockImplementation(mockApi({ payload: GET_ROLE_PROMISE }))`

or


`getRole.mockImplementation(mockApi({ errors: true }))`